packages/runtime/ui/src/base/snsIds.ts

```diff
@@ -0,0 +1,4 @@
+// TODO temporarily copy instead of reference
+
+export const MINDS_ID = 'minds.com'
```

Comment:
We need a better place for listing all supported SNS ids.

Reply:
Would do it in further refactoring.